src/nxdoc/nuxeo-server/installation/nuxeo-and-redis.md

```diff
@@ -173,16 +173,16 @@ By default Lock managed on documents are stored in the repository backend. When
 
 ## Clustering invalidation
 
-### Visual Content Store ([VCS](https://doc.nuxeo.com/nxdoc/vcs/)) Row Cache Invalidation
+### Visual Content Store ([VCS]({{page page='vcs'}})) Row Cache Invalidation
 
 Managing *VCS* (Meaning RDBMS) row cache invalidations with [Redis instead of using the database](https://jira.nuxeo.com/browse/NXP-14923) can improve performance on concurrent writes and provides synchronous invalidations.
 
-### Document Based Storage ([DBS](https://doc.nuxeo.com/nxdoc/dbs/)) Cache Invalidation
+### Document Based Storage ([DBS]({{page page='dbs'}})) Cache Invalidation
 
 For Nuxeo 8.10 and 9.10 the *DBS* layer (used to connect with MongoDB or Marklogic) [has a cache](https://jira.nuxeo.com/browse/NXP-20640) its invalidation in cluster mode requires Redis.
 
 {{#> callout type='warning' }}
-A Nuxeo No-Redis cluster configuration can be applied at certain conditions, see: [No-Redis Nuxeo Cluster Configuration](https://doc.nuxeo.com/nxdoc/kafka/#andquotno-redisandquot-nuxeo-cluster) for more info.
+A Nuxeo No-Redis cluster configuration can be applied at certain conditions, see: [No-Redis Nuxeo Cluster Configuration]({{page page='nxdoc/kafka/#andquotno-redisandquot-nuxeo-cluster'}}) for more info.
```

Comment:
You need `({{page page='nxdoc/kafka'}}#andquotno-redisandquot-nuxeo-cluster)` as the `page` stuff doesn't know to interpret anchors